client/ui/options.ts

```diff
@@ -35,7 +35,7 @@ export default class OptionsPanel extends TabbedModal {
 		this.renderHidden(hidden.size)
 		this.assignValues()
 
-		hook("renderOptionValue", ([id, type, val]) =>
+		hook("renderOptionValue", (id, type, val) =>
```

Comment:
This is a separate bug fix. It should be in it's own commit or PR in case this gets reverted.

Reply:
Sure.